package.json

```
@@ -14,28 +14,32 @@
     "url": "https://github.com/AnyVisionltd/anv-ui-components"
   },
   "dependencies": {
-    "@anyvision/style-guide": "1.0.4-beta.13",
-    "@anyvision/anv-icons": "^1.0.0-beta.27",
+    "@anyvision/anv-icons": "^2.0.0-beta.3",
```

```suggestion
    "@anyvision/anv-icons": "2.0.0-beta.3",
```
do "npm i" again after the change